doc/cephfs/fs-volumes.rst

```diff
@@ -87,6 +87,14 @@ Fetch the absolute path of a subvolume group using::
 
     $ ceph fs subvolumegroup getpath <vol_name> <group_name>
 
+Authorize cephx auth IDs, the read/read-write access to fs subvolume groups::
+
+    $ ceph fs subvolumegroup authorize <vol_name> <group_name> <auth_id> [--access_level=<access_level>]
```

Comment:
Mention what are the possible values of access_level option, r or rw.

Reply:
Done